chart/values.yaml

```diff
@@ -549,6 +555,36 @@ data:
   # Add custom annotations to the broker url secret
   brokerUrlSecretAnnotations: {}
 
+# SQLite shared file system configuration
+# This enables multiple Airflow pods to share a single SQLite database file
+# through a shared persistent volume. Suitable for development/testing environments.
+# WARNING: Not recommended for production use.
+sqliteShared:
+  # Enable SQLite shared mode
+  enabled: true
+
+  # Shared storage configuration
+  persistence:
+    # Enable persistent volume for SQLite database file
+    enabled: true
+    # Storage class name for the PVC (empty means use default storage class)
+    storageClassName: ""
+    # Access mode for the persistent volume
+    # Note: Local Path Provisioner (used in KinD) only supports ReadWriteOnce
+    # For production with multiple nodes, use ReadWriteMany with NFS or cloud RWX storage
+    accessMode: ReadWriteMany
+    # Size of the persistent volume
+    size: 1Gi
+    # Annotations for the PVC
+    annotations: {}
+    # Existing PVC name (optional, if you want to use an existing PVC)
+    existingClaim: ""
+
+  # Node selector for pod scheduling (used when access mode is ReadWriteOnce)
+  # Ensures all pods are scheduled on the same node to share the volume
+  nodeSelector: {}
```

Comment:
I do not think there is a need to complicate this with RWO and RWX. Let's just use PVC mode and not bother about node selector for now.